lib/generators/iiif_print/install_generator.rb

```diff
@@ -89,6 +89,11 @@ def add_faceted_attribute_decorator
       copy_file 'faceted_attribute_renderer_decorator.rb', 'app/renderers/hyrax/renderers/faceted_attribute_renderer_decorator.rb'
     end
 
+    def add_create_relationships_job_decorator
+      # supports setting default parent thumbnail to the first child's thumbnail, if one doesn't exist
+      copy_file 'create_relationships_job_decorator.rb', 'app/jobs/bulkrax/create_relationships_job_decorator.rb'
+    end
+
```

Comment:
I'm not sure why this diff is here, it's already in `main` so I would have expected this to not be a diff

Reply:
have you pulled `main` into this branch since the change was put into `main`?

Reply:
i have not, i can try that

Reply:
I went ahead and just removed it so now we should be good